examples/data-generator/src/customers.ts

```diff
@@ -16,11 +16,11 @@ export default (db, { serializeDate }) => {
         const last_name = name.lastName();
         const email = internet.email(first_name, last_name);
         const birthday = has_ordered ? date.past(60) : null;
-        const avatar = has_ordered
-            ? 'https://marmelab.com/posters/avatar-' +
-              numberOfCustomers +
-              '.jpeg'
-            : undefined;
+        const avatar = has_ordered ?
+            'https://marmelab.com/posters/avatar-' +
+            numberOfCustomers +
+            '.jpeg' :
```

Comment:
We should probably re-enable indentation of binary operators, looks like it's missing an indentation level here.